tests/test_domain_py.py

```diff
@@ -44,3 +46,44 @@ def test_function_signatures():
 
     rv = parse('func(a=[][, b=None])')
     assert text_type(rv) == u'a=[], [b=None]'
+
+@with_app('html', testroot='domain-py')
+def test_memory_address(app, status, warning):
+    app.builder.build_all()
+    g_refs = 0
+    for f in ['index.html','searchindex.js']:
+        t = (app.outdir / f).text()
+        refs = t.count("0x")
+        g_refs += refs
+        if refs>0:
+            print("A memory address was found %d times in the %s file." % (refs,f))
```

Comment:
Just use

```python
assert refs == 0, "A memory address was found %d times in the file %s." % (refs, f)
```

and delete `g_refs` — it will be much cleaner and simplier.